libs/designer-ui/src/lib/mixedinputeditor.tsx

```diff
@@ -0,0 +1,110 @@
+import type React from 'react';
+import { useEffect, useState } from 'react';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+import type { ChangeHandler, ValueSegment } from '@microsoft/logic-apps-shared';
+import { SchemaEditor } from './schemaeditor';
+import { FloatingActionMenuInputs } from './floatingactionmenu/floatingactionmenuinputs';
+
+interface MixedInputEditorProps {
+  initialValue: ValueSegment[];
+  onChange: ChangeHandler;
+  supportedTypes?: string[];
+  isRequestApiConnectionTrigger?: boolean;
+  label?: string;
+  readonly?: boolean;
+  useStaticInputs: boolean;
+}
+
+// Function to check if `initialValue` contains "useSchemaEditor": true
+const checkIsSchemaEditor = (initialValue: ValueSegment[]): boolean => {
+  return initialValue.some((segment) => {
+    try {
+      const parsedValue = JSON.parse(segment.value);
+      return parsedValue?.useSchemaEditor === true;
+    } catch (error) {
+      console.error('Error parsing value segment:', error);
+      return false;
+    }
+  });
+};
+
+// Function to add "useSchemaEditor": true if it's missing
+const addUseSchemaEditor = (value: ValueSegment[]): ValueSegment[] => {
+  const updatedValue = value.map((segment) => {
+    try {
+      const parsedValue = JSON.parse(segment.value);
+      if (parsedValue && !Object.prototype.hasOwnProperty.call(parsedValue, 'useSchemaEditor')) {
+        parsedValue.useSchemaEditor = true;
+        return {
+          ...segment,
+          value: JSON.stringify(parsedValue),
+        };
+      }
+      return segment;
+    } catch (error) {
+      console.error('Error parsing or updating value segment:', error);
+      return segment;
+    }
+  });
+
+  return updatedValue;
+};
+
+export const MixedInputEditor: React.FC<MixedInputEditorProps> = ({
+  initialValue,
+  onChange,
+  supportedTypes,
+  isRequestApiConnectionTrigger,
+  label,
+  readonly,
+  useStaticInputs,
+}) => {
+  const [isSchemaEditor, setIsSchemaEditor] = useState<boolean>(() => {
+    const storedToggleValue = localStorage.getItem('toggle');
```

Comment:
toggle is generic enough here where it might accidentally conflict with someone else trying to use a toggle at some point in local storage. I would do something like 

```typescript
const storedToggleValue = localStorage.getItem('MixedInputEditor_toggle');
```

or similar